server/conf/messages.en-US

```diff
@@ -166,3 +166,4 @@ validation.textTooLong=Must contain at most {0} characters.
 validation.textTooShort=Must contain at least {0} characters.
 validation.tooFewSelections=Please select at least {0}.
 validation.tooManySelections=Please select fewer than {0}.
+
```

Comment:
**suggestion (blocking):** add the english messages here as well

Reply:
If I add it here, the conf/Messagestests.java are failing. If anything is added to the ignore list, the tests expect that it only be present in conf/messages and not in any other translation files.